cmd/bb_worker/main.go

```diff
@@ -232,6 +235,19 @@ func main() {
 					log.Fatal("Failed to marshal worker ID: ", err)
 				}
 
+				var devices []filesystem.Device
+				for _, device := range configuration.Devices {
+					var stat unix.Stat_t
+					if err := unix.Stat(filepath.Join("/dev", device), &stat); err != nil {
+						log.Fatalf("Unable to stat device /dev/%#v", device)
+					}
+					devices = append(devices, filesystem.Device{
+						Name: device,
+						Rdev: stat.Rdev,
+						Mode: os.FileMode(stat.Mode),
```

Comment:
Inheriting whether something is a block or character device and its rdev number make sense. I do wonder whether permissions should be inherited. That’s unrelated, right?